app/controllers/claims/support/support_users_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Claims::Support::SupportUsersController < Claims::Support::ApplicationController
+  before_action :set_support_user, only: %i[show]
+
+  def index
+    @support_users = Claims::SupportUser.all
+  end
+
+  def new
+    @support_user = if params[:support_user].present?
+                      Claims::SupportUser.new(support_user_params)
+                    else
+                      Claims::SupportUser.new
+                    end
+  end
+
+  def check
+    @support_user = if params[:id].present?
```

Comment:
Is this necessary? I can't see a case where `:id` would ever be passed to the `:check` method

Reply:
You're right. Sorry, this was a relic from the edit flow.